test_variables.env.example

```diff
@@ -2,18 +2,32 @@ DEVNET_PATH=/path/to/starknet-devnet-rs/target/release/starknet-devnet
 
 # Network tests
 NETWORK_TEST_MODE=disabled
-NETWORK_TEST_NETWORK_NAME=INTEGRATION
+NETWORK_TEST_NETWORK_NAME=GOERLI_INTEGRATION
 
-# Integration tests
-INTEGRATION_RPC_URL=http://example-node-url.com/rpc
-INTEGRATION_PRIVATE_KEY=0x123456789
-INTEGRATION_ACCOUNT_ADDRESS=0x123456789123456789123456789123456789123456789123456789123456789
-INTEGRATION_CONST_NONCE_PRIVATE_KEY=0x1a1a1a1a1a
-INTEGRATION_CONST_NONCE_ACCOUNT_ADDRESS=0x2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2
+# Goerli Integration tests
+GOERLI_INTEGRATION_RPC_URL=http://example-node-url.com/rpc
+GOERLI_INTEGRATION_PRIVATE_KEY=0x123456789
+GOERLI_INTEGRATION_ACCOUNT_ADDRESS=0x123456789123456789123456789123456789123456789123456789123456789
+GOERLI_INTEGRATION_CONST_NONCE_PRIVATE_KEY=0x1a1a1a1a1a
+GOERLI_INTEGRATION_CONST_NONCE_ACCOUNT_ADDRESS=0x2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2
 
-# Testnet config
+# Goerli Testnet config
 TESTNET_RPC_URL=http://example-node-url.com/rpc
 TESTNET_PRIVATE_KEY=0x123456789
 TESTNET_ACCOUNT_ADDRESS=0x123456789123456789123456789123456789123456789123456789123456789
 TESTNET_CONST_NONCE_PRIVATE_KEY=0x1a1a1a1a1a
 TESTNET_CONST_NONCE_ACCOUNT_ADDRESS=0x2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2b2
```

Comment:
Why these variables are without `GOERLI` prefix?

Reply:
Fixed, thank you!